projects/driver_controls/src/fsm/hazards_fsm.c

```diff
@@ -1,34 +1,32 @@
-#include "hazard_light_fsm.h"
+#include "hazards_fsm.h"
+#include "can_transmit.h"
 #include "event_arbiter.h"
+#include "exported_enums.h"
 #include "input_event.h"
-#include "log.h"
 
 // Hazard light FSM state definitions
-
-FSM_DECLARE_STATE(state_hazard_on);
 FSM_DECLARE_STATE(state_hazard_off);
+FSM_DECLARE_STATE(state_hazard_on);
 
 // Hazard light FSM transition table definitions
+FSM_STATE_TRANSITION(state_hazard_off) {
+  FSM_ADD_TRANSITION(INPUT_EVENT_HAZARD_LIGHT, state_hazard_on);
+}
 
 FSM_STATE_TRANSITION(state_hazard_on) {
   FSM_ADD_TRANSITION(INPUT_EVENT_HAZARD_LIGHT, state_hazard_off);
-}
 
-FSM_STATE_TRANSITION(state_hazard_off) {
-  FSM_ADD_TRANSITION(INPUT_EVENT_HAZARD_LIGHT, state_hazard_on);
+  FSM_ADD_TRANSITION(INPUT_EVENT_POWER, state_hazard_off);
```

Comment:
In normal cars hazards stay on regardless... What's the intent of turning them off for power or fault events?

Reply:
I thought rear still needed to be powered for strobe?

Reply:
Right, so to keep the hazards/turn signals consistent, I turn them off. Otherwise, we'd have the rear signals blinking while the front is all dead.